### DISCUSSION TITLE
Long term effects of TDCS/TMS in models

### DISCUSSION THREAD

**Discussion by michefabb** [simnibs/simnibs#420] [7.Creating head models] [answered]
Dear SimNIBS experts,

I am looking for references about how to implement  long term effects of TDCS and TMS in models, would you have any?

Appreciated!

Mic

**Comment by Kstoffer** [comment, marked as answer]
We do not support any modelling of this kind in simNIBS, actually we really only model the static field created by brain stimulation at the moment.
To the best of my knowledge the availability of models seeking to model long terms effects of brain stimulation is still very scarce, so I do not even know where to begin with this.
Best,
Kristoffer

**Reply by michefabb** [reply]
Thanks @Kstoffer , appreciated.

Mic
